owncloudApp/src/main/res/values/strings.xml

```diff
@@ -607,6 +610,7 @@
     <string name="prefs_logs_summary">Logs may contain sensitive information. Sharing logs with others is sole user responsibility</string>
 
     <string name="synchronizing_account">Synchronizing account</string>
-    <string name="prefs_enable_logging">Enable logging</string>
-    <string name="prefs_enable_logging_summary">When activated, logs may impact performance and include sensitive information. However the logs are not subject to automatic submission to ownCloud servers. Sharing logs with others is sole user responsibility.</string>
+    <string name="prefs_subsection_security_summary">Set locks to access the app and allow touches with other visible windows</string>
+    <string name="prefs_subsection_logging_summary">Keep track of logs produced by the execution of the app</string>
+    <string name="prefs_subsection_more_summary">App info, feedback and others</string>
```

Comment:
Group new `prefs_subsection` with the other `prefs` keys. I know this is a little messy, but the cleaner we leave it, the less work we will need to do in the future.